### PR TITLE
Add .net foundation announcement post

### PR DESCRIPTION
**Pull request by Cheesebaron**
✨ Whaaaat?! We are joining the .NET Foundation

This PR adds the blog post

### CODE REVIEW

**Comment by kjeremy** [comment]
Awwwwwww YEAH!

File: docs/_posts/2017-11-15-joining-net-foundation.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ - mvvmcross
+ - community
+---
+
+As part of the [.NET Summer Hackfest][hackfest] we announced that we were working on joining the [.NET Foundation][netfoundation]. We have just received confirmation that we were accepted! So we are very happy to announce that we are joining the .NET Foundation, which governs a lot of great open source projects which impact you daily.
+
+## What does this mean for MvvmCross?
+The .NET Foundation was founded as an independent organization to foster open source development and collaboration. The MvvmCross teams is all for that and we want to participate in not only making MvvmCross better, but strengthening the .NET ecosystem.
+
+Joining the .NET Foundation gives us a lot of benefits raning from legal help, project guidance and monitoring, marketing, monetary support and a bunch of greate technical support services that we would otherwise have to spend a lot of money on.
```
Comment (Plac3hold3r) [inline docs/_posts/2017-11-15-joining-net-foundation.md]: Should "raning" not be ranging?
Should "greate" not be great?

**Comment by Plac3hold3r** [comment]
Congrats to MvvmCross core team and all contributes!!

**Comment by Cheesebaron** [comment]
Thanks for pointing those out @Plac3hold3r 👍 My VS Code doesn't have spellcheck installed, should probably do that 😄

**Pull request merged**